### DISCUSSION TITLE
Dynamic Tag to Add the website to the Home Screen

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21271] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi

As you know, major browsers such as Chrome and Safari offer the add to home screen feature. By using this feature, users can quickly enter the website for the next time. Because a shortcut for the website is added on their mobile or desktop.

![add to home screen](https://user-images.githubusercontent.com/88034902/217680591-bb19f792-8104-4feb-a918-b874e19f0ae5.jpg)

This is a wonderful feature. But many users do not know about this feature. Or many users don't have the patience to add the website to the home screen. **Elementor can add this feature to make everything go great.**

### Describe the solution you'd like

Add a new dynamic tag **(add to home screen)**

we can use this dynamic tag for button or other widgets widget. **for example :**

After the user enters the website, a popup will be displayed. If the user click on the **add now!** button, the website will be added to the user's home screen immediately. 

![a](https://user-images.githubusercontent.com/88034902/217681253-b5c5df67-a3af-4ee0-92b2-de43c0152867.png)

This feature will be super useful and awesome. Especially for Store sites. This feature can bring a lot of ideas that I hope other users will think about.

**Comment by fabigrafo** [comment]
This is possible with every website on apple's iOS. And I think with android too.

**Reply by Michaelgimii** [reply]
Right, I said at the beginning that this feature exists in browsers. But by using this feature, everything will happen more easily. In fact, instead of users finding this option in their browser settings, we show it to them using Elementor. It will be added to home screen with just one click.
